packages/renderer/src/App.spec.ts

```diff
@@ -256,3 +259,91 @@ test('leaving Dashboard Page saves it in lastPage storage', async () => {
   await tick();
   expect(get(lastPage).name).equals('Dashboard Page');
 });
+
+describe('Table persistence functionality', () => {
+  test('should set tablePersistenceCallbacks store on app initialization', async () => {
+    // Mock the window methods
+    Object.defineProperty(window, 'loadLayoutConfig', { value: vi.fn() });
+    Object.defineProperty(window, 'saveLayoutConfig', { value: vi.fn() });
+    Object.defineProperty(window, 'resetLayoutConfig', { value: vi.fn() });
```

Comment:
should be automatically mocked by https://github.com/podman-desktop/podman-desktop/blob/main/packages/renderer/vite.tests.setup.js#L71